### PR TITLE
Adding NodeJS content back in

### PR DESCRIPTION
**Pull request by wordlee**
# About

This is a post about deploying applications written in Node.js with a database backend with Octopus Deploy.

# Pre-requisites

- [X] The draft is complete and this post is ready to be reviewed.
- [X] I'm familiar with the [writing for Octopus TL;DR](https://style.octopus.com/writing-at-octopus-tldr).
- [X] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [X] The images I've included follow the [rules for working with images](https://style.octopus.com/images#rules-for-screenshots).

### CODE REVIEW

File: blog/2021-05/deploying-nodejs/index.md [hunk 1]
```diff
@@ -0,0 +1,1507 @@
+---
+title: "Deloying a NodeJS application with Octopus Deploy"
+description: "Learn how to deploy an application written in NodeJS with a MongoDB backend with Octopus Deploy"
+author: shawn.sesna@octopus.com
+visibility: private
+published: 2022-04-05-1400 
+metaImage: 
+bannerImage: 
+tags:
+ -
```
Comment (wordlee) [inline blog/2021-05/deploying-nodejs/index.md:10]:
```suggestion
 - DevOps
```

**Comment by TeganAli** [comment]
@wordlee I will make more edits to the back ticks etc once you've reviewed the deploying PHP post.

File: blog/2021-05/deploying-nodejs/index.md [hunk 2]
```diff
@@ -0,0 +1,1531 @@
...
+- Octopus Deploy: Push packages
+- Octopus Deploy: Octopus Build Information
+
+The first step is self-explanatory, so I won't detail that one.
+
+### npm
+Bamboo has a built-in step that will execute `npm` commands.  Select that step and enter `install` as the `Command` to run.
+
+:::warning
+Though the step exists in Bamboo, you still need to install the Node `Executable` on the build server and define its location in **{{Bamboo administration > Executables}}**.
```
Comment (TeganAli) [inline blog/2021-05/deploying-nodejs/index.md:1357]: @wordlee I added the syntax you showed me Tuesday, have I done it correctly here? (The one that creates a bolded string of text with an arrow where the greater than sign is.)

File: blog/2021-05/deploying-nodejs/index.md [hunk 3]
```diff
@@ -0,0 +1,1531 @@
...
+The previous step of `npm install` will download and install the dependencies defined in `package.json` file of our project into a node_modules folder.  With all necessary modules installed, we can package the application to be deployed:
+
+- PackageId: `BestBags.Web`
+- Version number: `${bamboo.inject.buildVersion}`
+- Package format: zip
+- Package base folder: `${bamboo.build.working.directory}`
+- Package output folder: `${bamboo.build.working.directory}\artifacts`
+
+:::warning
+Similar to npm, you will need to install the Octopus Deploy CLI on the build server and define its location in **{{Bamboo administration > Executables}}**
```
Comment (TeganAli) [inline blog/2021-05/deploying-nodejs/index.md:1385]: @wordlee I added the syntax you showed me Tuesday. Have I done it correctly here? (The one that creates a bolded string of text with an arrow where the greater than sign is.)

**Pull request merged**